fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -23,10 +23,10 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :command,
                                        env_name: "FL_CARTHAGE_COMMAND",
-                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`, `archive`)",
                                        default_value: 'bootstrap',
                                        verify_block: proc do |value|
-                                         raise "Please pass a valid command. Use one of the following: build, bootstrap, update" unless %w(build bootstrap update).include? value
```

Comment:
Whoops this comment is on the old commit. Moving relevant bits to current commit!